Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -238,170 +245,92 @@ Plot PD marker against concentration. Do you see any relationship? Does response
 ```{r, cache = TRUE, fig.width = 6, fig.height = 4, warning = FALSE, message= FALSE}
 
 data_to_plot <- list()
-data_to_plot[[1]] <- my.data[my.data$CMT==2&my.data$TRTACT!="Placebo"&my.data$PROFDAY==5,
-                             c("TIME","LIDV","ID","TRTACT","CENS")]
+data_to_plot[[1]] <- pk_data[pk_data$TRTACT_low2high!="Placebo"&pk_data$PROFDAY==5,
+                             c("TIME","LIDV","ID","TRTACT_low2high","CENS" )]
 data_to_plot[[1]]$Concentration <- data_to_plot[[1]]$LIDV
 data_to_plot[[1]]$LIDV <- NULL
 
-data_to_plot[[2]] <- my.data[my.data$CMT==3&my.data$PROFDAY==5,
-                             c("TIME","LIDV","ID","TRTACT")]
+data_to_plot[[2]] <- pd_data[pd_data$PROFDAY==5,
+                             c("TIME","LIDV","ID","TRTACT_low2high")]
 data_to_plot[[2]]$Response <- data_to_plot[[2]]$LIDV
 data_to_plot[[2]]$LIDV <- NULL
 
-data_to_plot2 <- merge(data_to_plot[[1]],data_to_plot[[2]],by=c("TIME","ID","TRTACT"))
+data_to_plot2 <- merge(data_to_plot[[1]],data_to_plot[[2]],by=c("TIME","ID","TRTACT_low2high"))
 
-data_to_plot2$TRTACT <- factor(data_to_plot2$TRTACT, levels = rev(levels(data_to_plot2$TRTACT)))
+data_to_plot2$TRTACT_low2high <- factor(data_to_plot2$TRTACT_low2high, levels = rev(levels(data_to_plot2$TRTACT_low2high)))
 
```

Comment:
Lines 248 through 260 can be deleted, and line 262 replaced with:
gg <- ggplot(data = pkpd_data_wide  %>% subset(PROFDAY %in% 5) , aes(x = CONC, y = RESPONSE))